visualizeInput.py

```diff
@@ -0,0 +1,31 @@
+# To visualise the data from Hacktoberfest_Inputt, we plot the occcurences of 
+# each relation
+
+import pandas as pd
+import matplotlib.pyplot as plt
+from pprint import pprint
+from collections import Counter
+
+df = pd.read_excel('Hacktoberfest_Inputt.xlsx').to_dict()
+
+rows = list(df['Source Color'].keys())
+
+valuesSc = list(df['Source Color'].values())    #Source colors
+valuesDc = list(df['Dest color'].values())      #Destination colors
+
+# Zone relations
+relations = []
+
+for i in rows:
+    relations.append((str(valuesSc[i])+' -> '+str(valuesDc[i])))
+# print(relations)
+
+# Associating the relations with their occurences
+relationFlow = Counter(relations)
+# pprint(relationFlow)
+
+# A scatter plot to show the occurences
+x,y = zip(*relationFlow.items())
+plt.scatter(x,y)
+# plt.plot(x,y)
+plt.show()
```

Comment:
```
if __name__== "__main__":
    visualize()
```
Make it a module

Reply:
line #29

- Rather than `plt.scatter` , I like `plt.bar`
- https://python-graph-gallery.com/320-basic-network-from-pandas-data-frame/ is the best!! Might be daunting, if you interested raise a 2nd PR for this visualization only and work on the changes mentioned for the current PR :)

Reply:
You're right bro, this is the best way to go👌. I'll add this network and comment out the scatter just in case , if we need it in the future.

Reply:
So, I'll close this PR and raise a new one?

Reply:
> So, I'll close this PR and raise a new one?

If you think you can make the plot like this network chart that'll be superb! Since, that task is little difficult, I was suggesting if you can make the plot in the network fashion then raise a new PR. Keep this PR for bar plot/scatter plot :)